### ISSUE TITLE
shape 图片的通道

### ISSUE THREAD

**Issue by bryanduran99** [jiawei151/SGIEL_VIReID#5]
您好，请问下您生成的SHAPE 图片是单通道的该怎么处理呢， 直接广播成三个通道吗？

**Comment by bryanduran99** [comment]
奥，我理解过了，应该是生成head, torso, or limbs 三个的logits 组成三个通道

**Comment by bryanduran99** [comment]
limbs 是这几个 'Upper Arms', 'Lower Arms', 'Upper Legs', 'Lower Legs' logits 直接求和吗

**Comment by jiawei151** [comment]
您好，是广播成三个通道；属于身体部位的都直接求和；我们刚刚上传了一个链接您可以下载看下
